src/longbow/sift/command.py

```diff
@@ -0,0 +1,208 @@
+import json
+import logging
+import time
+import os
+import sys
+
+import click
+import click_log
+
+import tqdm
+import pysam
+from construct import *
+from collections import Counter
+
+import longbow.utils.constants
+from ..utils import bam_utils
+from ..utils import model as LongbowModel
+from ..utils.model import LibraryModel
+from ..annotate.command import get_segments
+
+logging.basicConfig(stream=sys.stderr)
+logger = logging.getLogger("sift")
+click_log.basic_config(logger)
+
+
+@click.command(name=logger.name)
+@click_log.simple_verbosity_option(logger)
+@click.option(
+    "-p",
+    "--pbi",
+    required=False,
+    type=click.Path(exists=True),
+    help="BAM .pbi index file",
+)
+@click.option(
+    "-o",
+    "--output-bam",
+    default="-",
+    type=click.Path(exists=False),
+    help="filtered bam output (passing reads only)  [default: stdout]",
+)
+@click.option(
+    "-x",
+    "--reject-bam",
+    default="/dev/null",
+    type=click.Path(exists=False),
+    help="Filtered bam output (failing reads only)  [default: /dev/null]",
+)
+@click.option(
+    "-m",
+    "--model",
+    help="The model to use for annotation.  If not specified, it will be autodetected from "
+         "the BAM header.  If the given value is a pre-configured model name, then that "
+         "model will be used.  Otherwise, the given value will be treated as a file name "
+         "and Longbow will attempt to read in the file and create a LibraryModel from it.  "
+         "Longbow will assume the contents are the configuration of a LibraryModel as per "
+         "LibraryModel.to_json()."
+)
+@click.option(
```

Comment:
This option `-v` masks the logging verbosity argument.  It needs to be changed to something different.

Reply:
Changed to `-l` / `--validation-model`